sdk/src/test/java/io/radar/sdk/RadarTest.kt

```diff
@@ -1273,4 +1274,18 @@ class RadarTest {
         assertRoutesOk(callbackRoutes)
     }
 
+    @Test
+    fun test_Radar_trackingOptionsFallback_success() {
+        assertEquals(null, RadarSettings.getFallbackTrackingOptions(context))
+        RadarSettings.setTrackingOptions(context, RadarTrackingOptions.CONTINUOUS)
+        assertEquals(RadarTrackingOptions.CONTINUOUS, RadarSettings.getTrackingOptions(context))
+        RadarSettings.revertToFallbackTrackingOptions(context)
+        assertEquals(RadarTrackingOptions.CONTINUOUS, RadarSettings.getTrackingOptions(context))
+
+        RadarSettings.setFallbackTrackingOptions(context, RadarTrackingOptions.RESPONSIVE)
+        assertEquals(RadarTrackingOptions.RESPONSIVE, RadarSettings.getFallbackTrackingOptions(context))
+        RadarSettings.revertToFallbackTrackingOptions(context)
+        assertEquals(RadarTrackingOptions.RESPONSIVE, RadarSettings.getTrackingOptions(context))
+    }
```

Comment:
Can you add the case where the sdk calls `setTrackingOptions` then calls `revertToFallbackTrackingOptions`?

Reply:
good idea

Reply:
actually, the latest version does this